esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -898,6 +898,28 @@ void WsWuInfo::getGraphTimingData(IArrayOf<IConstECLTimingData> &timingData)
         it->playProperties(visitor);
 }
 
+void WsWuInfo::getServiceNames(IEspECLWorkunit &info, unsigned long flags)
+{
+    if (!(flags & WUINFO_IncludeServiceNames))
+        return;
+
+    StringArray serviceNames;
+    WuScopeFilter filter("depth[1,]");
+    filter.sourceFlags = SSFsearchGraph;
```

Comment:
Only the public member functions should be used.  I will investigate why those member variables are not private

Reply:
Ok